SU-SWS/stanford_media
# updated entity usage link due to new column names in v2 release

# READY FOR REVIEW

# Summary
- Updated columns names with the update in entity usage.

# Needed By (Date)
- asap

# Urgency
- low

# Steps to Test

1. use the branch and update composer
2. view the media usage page at `/admin/content/media/{mid}`
3. verify no errors occur.

# Affected Projects or Products
- MRC
- Arch


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
